retrieval/discovery/marathon.go

```diff
@@ -0,0 +1,177 @@
+package discovery
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	clientmodel "github.com/prometheus/client_golang/model"
+
+	"github.com/prometheus/log"
+	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/retrieval/discovery/marathon"
+)
+
+const (
+	// ImageLabel is the label that is used for the docker image running the service
+	ImageLabel clientmodel.LabelName = "image"
```

Comment:
I see you already created a sub-package for some functionality. I guess moving everything in there makes sense.